gen_license.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2013 The lime Authors.
```

Comment:
Not sure I like that this file will end up duplicated across all the projects...